packages/storage/src/indexed-db/indexed-db.test.ts

```diff
@@ -565,67 +562,6 @@ describe('IndexedDb', () => {
     });
   });
 
-  describe('epochs', () => {
```

Comment:
thought: makes me uneasy removing tests without a replacement. However, maybe it's fine in this case.

Reply:
I think it's alright since I opted for a more minimal diff.